aws/rust-runtime/aws-inlineable/src/presigning.rs

```diff
@@ -147,7 +147,10 @@ impl PresigningConfigBuilder {
             return Err(ErrorKind::ExpiresInDurationTooLong.into());
         }
         Ok(PresigningConfig {
-            start_time: self.start_time.unwrap_or_else(SystemTime::now),
+            start_time: self.start_time.unwrap_or_else(
+                #[allow(clippy::disallowed_methods)]
+                SystemTime::now,
```

Comment:
Should we make these `Option` in the `PresigningConfig` and resolve to a default using `TimeSource` where they're used?

Reply:
I considered but I think these are actually used in customer code right?

Reply:
I believe they're only given in the builder when requesting a presigned URL.